src/cryptography/hazmat/backends/openssl/backend.py

```diff
@@ -1369,7 +1369,7 @@ def load_pem_x509_certificate(self, data):
             self._consume_errors()
             raise ValueError(
                 "Unable to load certificate. See https://cryptography.io/en/la"
-                "test/faq/#why-can-t-i-import-my-pem-file for more details."
+                "test/faq.html#why-can-t-i-import-my-pem-file for more details."
```

Comment:
All these lines exceed the char limit now so flake8 fails. 😢

Reply:
Reminds me that I opened the failing travis result in a tab to look at what exactly was failing... Will fix in a minute!